src/app/cli/src/coda.ml

```diff
@@ -194,6 +198,16 @@ let daemon logger =
            "/ip4/IPADDR/tcp/PORT/ipfs/PEERID initial \"bootstrap\" peers for \
             discovery"
          (listed string)
+     and genesis_runtime_constants =
+       flag "genesis-constants"
+         ~doc:
+           (sprintf
+              "PATH path to the runtime-configurable constants. For example: \
+               %s  (default: compiled constants)"
```

Comment:
I would move the default description to the end of the first sentence, where it will be easier to notice.